### ISSUE TITLE
Background images missing on website science page and items order is wrong

### ISSUE THREAD

**Issue by blushi** [regen-network/regen-web#834]
**Describe the bug**
The background images for the ecological outcomes are missing on the website science page. The order seems off as well.

**To Reproduce**
1. Go to https://regen.network/science
2. Scroll down to "ECOLOGICAL OUTCOMES"

**Expected behavior**
![image](https://user-images.githubusercontent.com/3149358/149895926-617b56b9-6273-4740-ae1c-6a6e6149bd34.png)

**Screenshots**
![image (1)](https://user-images.githubusercontent.com/3149358/149895940-455485b8-456c-4036-97ce-d543569e123f.png)

**Additional context**
This is happening after #797

cc/ @haveanicedavid 
____

## For Admin Use

- [x] Not duplicate issue
- [x] Appropriate labels and zenhub epics applied
- [x] Appropriate contributors tagged

**Comment by haveanicedavid** [comment]
This is fixed now, but I don't have any PR to map to it. 

Basically I deleted these images on accident during the migration not realizing they were still being referenced here.

I did discover another bug though - there's something going on with Gatsby caching sanity assets. I added the images in sanity and they showed locally, but only rendered correctly on the live site after I merged another PR (which kicked off a new build)
